.gitignore

```diff
@@ -1,24 +1 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
-*.o
-*.a
-*.so
-
-# Folders
-_obj
-_test
-
-# Architecture specific extensions/prefixes
-*.[568vq]
-[568vq].out
```

Comment:
are these no need to be there?

Reply:
yes. Looks like some one copied from some other place